### PR TITLE
Add /orchestrate skill for multi-task delegation

### PR DESCRIPTION
**Pull request by lpendrake**
## Description

Adds the `/orchestrate` skill — an opus-driven pattern for implementing complex, multi-part tasks. The orchestrator plans, delegates to sonnet/haiku agents in parallel worktrees, reviews their output, merges results, resolves conflicts (also via delegation), and ships one cohesive PR.

### Details

Born from direct experience orchestrating Sub-Epic 6 (#185) — 8 parallel documentation agents across isolated worktrees, all merged into a single PR.

**The skill covers:**
- **When to use** — tasks spanning 3+ files/systems, epics with sub-tasks
- **6 phases**: Understand → Plan → Execute → Review → Merge → Ship
- **Task design** — self-contained, one output area, testable in isolation
- **Batch ordering** — dependency-aware parallelism with clear rules
- **Conflict resolution** — delegated to sonnet agents with full context about both sides
- **Rogue agent detection** — what to watch for and how to recover
- **Non-negotiable rules** — never write code, never skip hooks, review every diff
- **Worktree cleanup** — stale worktrees pollute test runs (learned the hard way)

**Key value proposition for the user:**
- Fewer tickets to manage (opus holds the full context)
- Fewer bugs (unified review catches cross-cutting issues)
- Less overhead (no manual task orchestration breaking up context)
- Bigger but more cohesive PRs

### Manual Test Cases

**New Behavior**
- [x] `/orchestrate` appears in Claude Code's available skills list
- [x] Skill description is clear and actionable
- [x] All 6 phases are documented with concrete guidance

**Regression Checks**
- [x] All 1182 tests pass
- [x] No production code changed — skill file only

https://claude.ai/code/session_01E6u4uUNgJD3HwLx5yPMJhL

### CODE REVIEW

File: .claude/skills/orchestrate/SKILL.md [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+- What the project is and what it does (one sentence)
+- What specific change to make and why
+- Which files to read first for context
+- Which files to create or modify
+- The exact output file path(s)
+- Relevant conventions (from AGENTS.md, CLAUDE.md)
+- Commit message format: start with `#<issue-number>`, describe the
+  user-facing change (see CLAUDE.md Git etiquette). Instruct the
+  agent to squash its work into a single commit before finishing.
+- Any patterns to follow (point at existing code as examples)
```
Comment (lpendrake) [inline .claude/skills/orchestrate/SKILL.md:134]: It's verbose but I've always been a fan of providing a table of test (unit or integration) scenarios that must be implemented alongside the task.
number | test title | intended coverage |
1 | "foo should verify X by doing Y" | checks that the code handles it when the caller passes in bla bla |
2 | ... | ... | 

Do you think it's worth the extra tokens both to generate and per task and for the implementing model to read and do or is it too verbose?

Comment (lpendrake) [reply .claude/skills/orchestrate/SKILL.md:134]: Worth it for code tasks. The orchestrator has the full cross-cutting context that the implementing agent lacks, so it's in the best position to specify meaningful edge cases. Without the table, agents tend to write tests that mirror the implementation rather than testing behavior.

I'll add it as an optional item in the prompt checklist — only for tasks that produce testable code, not docs/config. Pushing the update now.

---
_Generated by [Claude Code](https://claude.ai/code)_
